synced_versions_formulae.json

```diff
@@ -34,7 +34,6 @@
   ["libnetworkit", "networkit"],
   ["libnghttp2", "nghttp2"],
   ["libngspice", "ngspice"],
-  ["libqalculate", "qalculate-gtk", "qalculate-qt"],
```

Comment:
Seems like we still want `qalculate-gtk` and `qalculate-qt` in sync?

Reply:
yeah, that is true. let me update it.